Dockerfile

```diff
@@ -23,8 +31,7 @@ COPY scripts/.zshrc /root/.zshrc
 
 # build Haskell dependencies
 COPY Makefile cabal.project flora.cabal ./
-RUN cabal update
-RUN cabal build --only-dependencies -j8
+RUN cabal build --only-dependencies -j20
```

Comment:
We might want to leave the `-j` option without the number, few people have 20 cores on their machines :)